src/components/Footer.astro

```diff
@@ -0,0 +1,68 @@
+<footer>
+  <section class="small-body">
+    <div class="company">
+      <img src="img/Logo_black.svg" alt="Zasuń" class="footer_logo" />
+      <p>Piotr Zasuń</p>
+      <p>NIP: 626 269 96 59</p>
+    </div>
+    <div class="other">
+      <p>Copyright &copy 2023 Wszelkie prawa zastrzeżone</p>
+      <p>
+        Projekt i wykonanie: <a href="mailto:mateuszkarcz88@gmail.com"
+          >Unicorn Studio</a
+        >
+      </p>
+    </div>
+  </section>
+</footer>
+
+<style lang="scss">
+  footer {
+    margin-top: auto;
+    border-top: 1px solid var(--footer-color);
+
+    section {
+      margin: 40px 0;
+      gap: 16px;
+      display: block;
+
+      p {
+        color: var(--footer-color);
+      }
+    }
+
+    .small-body {
+      margin-left: 0;
+    }
+
+    img {
+      height: 14px;
+    }
+
+    .company {
+      display: flex;
+      gap: 8px;
+      align-items: center;
+    }
+
+    .other {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    a {
+      --footer-link-color: var(--black);
+
+      color: var(--footer-link-color);
+      font-weight: 400;
+
+      &::after {
+        background-color: var(--footer-link-color);
+      }
+
+      &:hover {
+        color: var(--footer-link-color);
```

Comment:
this might be removed, but to be confirm if the variable `--link-color` shoudnt' be redefined in here